modules/twinklebatchdelete.js

```diff
@@ -171,6 +171,10 @@ Twinkle.batchdelete.callback = function twinklebatchdeleteCallback() {
 		pages = pages.filter(function(page) {
 			return !page.missing && page.imagerepository !== 'shared';
 		});
+		// json formatversion=2 doesn't sort pages by namespace
+		pages = pages.sort(function(one, two) {
+			return one.ns - two.ns || one.title > two.title;
```

Comment:
```suggestion
			return one.ns - two.ns || (one.title > two.title ? 1 : -1);
```

sort's compareFn should return a number

Reply:
Word.  Done.